login/profile.go

```diff
@@ -113,6 +113,9 @@ func (userProfileClient *KeycloakUserProfileClient) Update(keycloakUserProfile *
 			// Observed that a 500 is returned whenever username/email is not unique
 			return errors.NewBadParameterError("username or email", fmt.Sprintf("%s , %s", *keycloakUserProfile.Email, *keycloakUserProfile.Username))
 		}
+		if resp.StatusCode == 403 || resp.StatusCode == 400 {
```

Comment:
https://github.com/almighty/almighty-core/issues/1308#issuecomment-303392041
Thanks,
mapping 403 is not needed in my case.